Core/WKWebViewConfigurationExtension.swift

```diff
@@ -48,6 +48,14 @@ extension WKWebViewConfiguration {
 
 fileprivate class Loader {
     
+    // Need to refactor so that that easylist and disconnect use the cache here instead of in their loader/parsers
```

Comment:
Can we put a TODO in front so the IDE flags it and a link to the Asana ticket scheduling it so it's not forgotten?

Reply:
Yep, was unsure if you were OK with TODOs being there.

Reply:
Preferably not - but this is comment is a todo so we may as well label it as such